### PR TITLE
Bump Docs to 7.0.0

### PR DESCRIPTION
**Pull request by Inscrutable**
Also extend copyright to 2018

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/plugin/debugging.rst [hunk 1]
```diff
@@ -54,7 +54,7 @@ adding the plugin project to Sponge's classpath will rid you of the necessity to
 file into your server directory after every change.
 
 First you need to ensure that you have your Run/Debug Configuration(s) set appropriately, as shown in the Sponge
-`README.md <https://github.com/SpongePowered/SpongeForge/blob/stable-6/README.md#Running>`_
+`README.md <https://github.com/SpongePowered/SpongeForge/blob/stable-7/README.md#Running>`_
```
Comment (ST-DDT) [inline source/plugin/debugging.rst]: Note to self: Check whether this needs to be `#running`.

**Comment by Inscrutable** [comment]
OK, I think that's all we need to at least bump the version to API 7 and let us get started. There will be a lot of files that need tinkering (especially in *Plugins*). I haven't tweaked the descriptions for the ``global.conf`` settings either, a few defaults changed and asynch lighting got threads. Let me know, or add a commit if I missed anything important.

File: source/plugin/debugging.rst (hunk 1 above)
Comment (ST-DDT) [reply source/plugin/debugging.rst]: Yeah, the hook needs to be lowercased.

**Pull request merged**